packages/manager/modules/pci/src/components/users/download-rclone/download-rclone.html

```diff
@@ -45,6 +64,12 @@
                 data-match="label"
             >
             </oui-select>
+            <small>
```

Comment:
**question:** Couldn't the `help-text` attribute be used on `<oui-field>` component?
see: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-components-field-webcomponent--help-text

Reply:
Hello Antoine, I wanted to add it conditionally that is only for Swift Containers. Thats why used span here.